packages/components/src/local/coa-status-board-2/__snapshots__/index.spec.tsx.snap

```diff
@@ -329,45 +326,14 @@ Array [
           role="rowgroup"
         >
           <div
-            className="sc-iwsKbI joYMLE rdt_TableRow"
+            className="sc-iwsKbI ftNEpz rdt_TableRow"
             id="row-Blue-3"
             onClick={[Function]}
             onDoubleClick={[Function]}
             role="row"
             selected={false}
             style={Object {}}
           >
```

Comment:
@lebaphi - I'm nervous of why these blocks of HTML in the snapshot have been removed.

Have we lost the expand/collapse buttons?

Reply:
Hi @IanMayo , I just want to make the new table has the same UI with the old table, just click on the row and it will expand the message.